### DISCUSSION TITLE
Sharing: World Time Zones Overlay

### DISCUSSION THREAD

**Discussion by thompehw** [osmandapp/OsmAnd#14695] [Show and tell]
In addition to the request and contributions at #13648, here is a nice raster tile map for OsmAnd that I made for my own use:
[TimeZones.zip](https://github.com/osmandapp/OsmAnd/files/9001038/TimeZones.zip)
(Click on the .sqlitedb file in a file manager and select OsmAnd to import it.)

It is taken from the [CIA World Factbook](https://www.cia.gov/the-world-factbook/maps/world-regional/) and [appears to be free to share](https://www.cia.gov/the-world-factbook/about/faqs/) even in modified form (USA public domain); a version of it is [in wikimedia](https://commons.wikimedia.org/wiki/File:World_Time_Zones_Map.png).

The original map is not centred on 0 degrees so I had to cut off the right end of the map (beyond 180) and attach it to the left. Then I georeferenced it in QGIS using many reference points - the projection is very different especially in the north. I generated tiles in QGIS then made the sqlitedb with MOBAC.

I'd be happy to fix any defects or share my intermediate files. The tiles are of size 256 at zoom 2-5 and are JPEG of quality 75. I did experiment with larger tiles that look better at low zoom when the map magnifier is used, but it didn't seem to be worth it: if something is unclear you can simply zoom-in.

I hope someone likes it! Here's a screenshot of how it looks in OsmAnd at z2 on a tablet:
![Screenshot_OsmAnd_TimeZones](https://user-images.githubusercontent.com/39751382/176178529-0f4c9710-4a1a-48cd-b4cc-2eee8a623013.jpg)

**Comment by karl-01** [comment]
Thank you for the World Time Zones.
I use it on a phone and so it is not such impressive as on your tablet, but it gives me all information I need.

Karl
